internal/cmd/local/command_kafka_start.go

```diff
@@ -188,3 +194,20 @@ func getContainerEnvironmentWithPorts(ports *v1.LocalPorts) []string {
 		fmt.Sprintf("KAFKA_REST_LISTENERS=http://0.0.0.0:%s", ports.KafkaRestPort),
 	}
 }
+
+func checkMachineArch() error {
+	cmd := exec.Command("uname", "-m")
```

Comment:
What does -m mean? Can you document that in a comment?

Reply:
`uname` outputs sys info, `-m` is specifically for arch.
putting it in comment